CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1001,26 +1071,31 @@ public let DECL_NODES: [Node] = [
         name: "Attributes",
         kind: .collection(kind: "AttributeList", collectionElementName: "Attribute"),
         nameForDiagnostics: "attributes",
+        description: "Attributes attached to the import declaration.",
         isOptional: true
       ),
       Child(
         name: "Modifiers",
         kind: .collection(kind: "ModifierList", collectionElementName: "Modifier"),
         nameForDiagnostics: "modifiers",
+        description: "Modifiers attached to the import declaration",
         isOptional: true
       ),
       Child(
         name: "ImportTok",
-        kind: .token(choices: [.keyword(text: "import")])
+        kind: .token(choices: [.keyword(text: "import")]),
+        description: "The `import` keyword for this declaration."
       ),
       Child(
         name: "ImportKind",
         kind: .token(choices: [.keyword(text: "typealias"), .keyword(text: "struct"), .keyword(text: "class"), .keyword(text: "enum"), .keyword(text: "protocol"), .keyword(text: "var"), .keyword(text: "let"), .keyword(text: "func"), .keyword(text: "inout")]),
+        description: "The kind of declaration being imported.",
         isOptional: true
       ),
       Child(
         name: "Path",
-        kind: .collection(kind: "AccessPath", collectionElementName: "PathComponent")
+        kind: .collection(kind: "AccessPath", collectionElementName: "PathComponent"),
+        description: "The access path to the module, submodule or symbol being imported."
```

Comment:
We’re not really happy with the name `AccessPath` here. I would just cut the word `access` here.

https://github.com/apple/swift-syntax/blob/2179d9fe6335c54e2fb13419c1186ac549d407c2/CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift#L33

I filed https://github.com/apple/swift-syntax/issues/1632 to rename the type.